src/utringbuffer.h

```diff
@@ -33,8 +33,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "utarray.h"  // for "UT_icd"
 
 typedef struct {
-    unsigned i;       /* index of next available slot; wraps at n */
-    unsigned n;       /* capacity */
+    size_t i;         /* index of next available slot; wraps at n */
+    size_t n;         /* capacity */
```

Comment:
Changing this from 32-bit to 64-bit makes the struct twice as large for no apparent benefit. (Also, it would break binary compatibility, although I'm not sure that's important in the case of utringbuffer.)